### PR TITLE
Preventing Notice in Shipping Zones

### PR DESCRIPTION
**Pull request by ahmedofali**
Adding variable to prevent the notice of undefined variable if function call did not made it through the switch case

### CODE REVIEW

File: includes/class-wc-shipping-zones.php [hunk 1]
```diff
@@ -55,6 +55,8 @@ public static function get_zone( $zone_id ) {
 	 * @return WC_Shipping_Zone|bool
 	 */
 	public static function get_zone_by( $by = 'zone_id', $id = 0 ) {
+                $zone_id = false ;
```
Comment (claudiosanches) [inline includes/class-wc-shipping-zones.php]: Need fix the indentation here, we use spaces on WordPress and not spaces.
And need to remove that extra space before `;`.

File: includes/class-wc-shipping-zones.php [hunk 2]
```diff
@@ -55,6 +55,8 @@ public static function get_zone( $zone_id ) {
...
+
```
Comment (claudiosanches) [inline includes/class-wc-shipping-zones.php]: Need remove spaces from this new line.

**Review by claudiosanches — CHANGES_REQUESTED** [review]
I recommend you install EditorConfig in your Editor/IDE, so this formatting issues will be easily avoided.

Please let me know when you update and fix the things I pointed out.
Thanks for your contribution.

**Comment by ahmedofali** [comment]
Well I have fixed indentation problem.

File: includes/class-wc-shipping-zones.php [hunk 3]
```diff
@@ -55,6 +55,8 @@ public static function get_zone( $zone_id ) {
 	 * @return WC_Shipping_Zone|bool
 	 */
 	public static function get_zone_by( $by = 'zone_id', $id = 0 ) {
+		$zone_id = false;
+
```
Comment (claudiosanches) [inline includes/class-wc-shipping-zones.php]: Still have spaces on this line.

**Pull request closed without merging**